sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -551,6 +586,504 @@ Parameters:
+          target AccessTarget = target::device>
+class dynamic_accessor {
+public:
+  template <AllocatorT>
+  dynamic_accessor(
+        command_graph<graph_state::modifiable> &graph,
+        buffer<DataT, 1, AllocatorT> &bufferRef,
+        const property_list &propList = {})
+  }
+
+  template <AllocatorT>
+  dynamic_accessor(
+        command_graph<graph_state::modifiable> &graph,
+        buffer<DataT, Dimensions, AllocatorT> &bufferRef,
+        const property_list &propList = {})
+
+  template <typename AllocatorT, typename TagT>
+  dynamic_accessor(
+        command_graph<graph_state::modifiable> &graph,
+        buffer<DataT, Dimensions, AllocatorT> &bufferRef,
+        TagT tag, const property_list &propList = {})
+
+  template <typename AllocatorT>
+  dynamic_accessor(
+        command_graph<graph_state::modifiable> &graph,
+        buffer<DataT, Dimensions, AllocatorT> &bufferRef,
+        range<Dimensions> accessRange,
+        const property_list &propList = {})
+
+  template <typename AllocatorT>
+  dynamic_accessor(
+        command_graph<graph_state::modifiable> &graph,
+        buffer<DataT, Dimensions, AllocatorT> &bufferRef,
+        range<Dimensions> accessRange,
+        id<Dimensions> accessOffset,
+        const property_list &propList = {})
+
+  template <typename AllocatorT>
+  dynamic_accessor(
+        command_graph<graph_state::modifiable> &graph,
+        buffer<DataT, Dimensions, AllocatorT> &bufferRef,
+        range<Dimensions> accessRange,
+        id<Dimensions> accessOffset,
+        const property_list &propList = {})
+
+  template <typename AllocatorT, typename TagT>
+  dynamic_accessor(
+        command_graph<graph_state::modifiable> &graph,
+        buffer<DataT, Dimensions, AllocatorT> &bufferRef,
+        range<Dimensions> accessRange,
+        id<Dimensions> accessOffset,
+        TagT tag, const property_list &propList = {})
+
+  template <typename AllocatorT>
+  void update(
+        buffer<DataT, Dimensions, AllocatorT> &newBufferRef);
+
+  template <typename AllocatorT>
+  void update(
+        buffer<DataT, Dimensions, AllocatorT> &newBufferRef,
+        range<Dimensions> accessRange);
+
+  template <typename AllocatorT>
+  void update(
+        buffer<DataT, Dimensions, AllocatorT> &newBufferRef,
+        range<Dimensions> accessRange,
+        id<Dimensions> accessOffset);
+};
+}
+----
+
+The `dynamic_accessor` class provides the {crs}[common reference semantics].
+
+:acbt: https://www.khronos.org/registry/SYCL/specs/sycl-2020/html/sycl-2020.html#sec:accessor.command.buffer.tags
+
+Table {counter: tableNumber}. Member functions of the `dynamic_accessor` class.
+[cols="2a,a"]
+|===
+|Member Function|Description
+
+|
+[source,c++]
+----
+template <AllocatorT>
+dynamic_accessor(
+      command_graph<graph_state::modifiable> &graph,
+      buffer<DataT, 1, AllocatorT> &bufferRef,
+      const property_list &propList = {})
+----
+|Available only when (Dimensions == 0).
```

Comment:
Could put backticks round this to match rendering in core spec
```suggestion
|Available only when `(Dimensions == 0)`.
```

(and other places)